.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -99,6 +99,7 @@ body:
         Which version of Apache DolphinScheduler are you running? We only accept bugs report from the LTS projects.
       options:
         - dev
+        - 3.0.1
         - 3.0.0
```

Comment:
maybe we should remove support of version 3.0.0, because 3.0.1 is a bugfix version of 3.0.0, so any issue exists in 3.0.0 should also in verison 3.0.1 or fix in version 3.0.1 already
```suggestion
```